src/components/Dashboard/Modals/AboutModal/AboutModal.tsx

```diff
@@ -50,6 +52,21 @@ export const AboutModal = ({ opened, closeModal, newVersionAvailable }: AboutMod
   const informations = useInformationTableItems(newVersionAvailable);
   const { t } = useTranslation(['common', 'layout/modals/about']);
 
+  const keybinds = [
```

Comment:
Is there no way to read this out dynamically?

Reply:
No